syntax = "proto3";

package response_models;

message GameOverResponseModel {}